pkg/configuration/endpoints/ip_blocks.go

```diff
@@ -0,0 +1,29 @@
+package endpoints
+
+import (
+	"github.com/np-guard/models/pkg/netset"
+)
+
+// a base struct to represent external endpoints, segments and rule block
+type ipBlock struct {
+	Block *netset.IPBlock
+}
+type RuleIPBlock struct {
+	ipBlock
+	origIP string
+	VMs    []EP
+}
+
+func NewRuleIPBlock(ip string, block *netset.IPBlock) *RuleIPBlock {
+	return &RuleIPBlock{origIP: ip, ipBlock: ipBlock{Block: block}}
+}
+
+type Segment struct {
```

Comment:
does it make sense to change this package name to `topology` instead of `endpoints`?
and also rename interface `EP` to `Endpoint` ?

Reply:
1. yes - segments and rule blocks are not endpoints. I didint dive into package structure, so didnt really know where to put them. changing package name is a good idea, please consider it when you reoerg pkg configuration.
2. Endpoint is a better name than EP, however, this interface is wildly used outside the package, as endpoints.EP.
choosing endpoints.Endpoint will make a lot of code lines longer, and harder to read. so I prefer EP